### DISCUSSION TITLE
[공지] 누리의 서재 기본 공지

### DISCUSSION THREAD

**Discussion by yeon-so** [nhnacademy-be7-nuri-books/nuri-books-demo#1] [공지사항]
# 기본 공지
 - 주 Scrum Master 교체
 - 매일 9시 10분 Scrum Meeting 진행
 
---

# 팀 규칙
  - 폴더 규칙
    - 계층(domain, repository, controller, exception, dto, service, common) > 도메인 > dto일 경우 response/request

**Comment by yongtaek2** [comment]
확인 완료!
